GenericNode/src/main/java/genericnode/Clients/TCPClient.java

```diff
@@ -0,0 +1,71 @@
+package genericnode.Clients;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.Socket;
+import java.util.AbstractMap.SimpleEntry;
+
+public class TCPClient extends Client {
+
+    Socket clientSocket;
+    DataOutputStream outToServer = null;
+    DataInputStream inFromServer = null;
+    @Override
+    public void put(String key, String value) throws IOException {
+        outToServer.writeUTF(key);
+        outToServer.writeUTF(value);
+        clientSocket.close();
```

Comment:
put command should output the result like:
$ java -jar GenericNode.jar tc localhost 1234 put a 123
server response:put key=a

Reply:
Got it. I will do the changes